crates/rpc/rpc-types-compat/src/engine/payload.rs

```diff
@@ -83,6 +83,7 @@ pub fn try_payload_v1_to_block<T: Decodable2718>(
         extra_data: payload.extra_data,
         // Defaults
         ommers_hash: EMPTY_OMMER_ROOT_HASH,
+        #[allow(deprecated)]
         difficulty: Default::default(),
```

Comment:
this is the header difficulty?
this shouldn't be deprecated?

Reply:
oh shoot